documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -293,34 +391,45 @@ You can find out what the plot titles (i.e. axes' labels etc.) of an object are
    Output=plot_titles(obj);
 
 
-The output returned provides (if such information exists) a vector ``[title_main, title_pax, title_iax, display_pax, display_iax, energy_axis]`` where ``title_main`` is the title that would appear at the top of the plot, ``title_pax`` contains the annotations for each of the plot axes, ``title_iax`` contains the legend detailing the integration axes limits etc, ``display_pax`` is a cell array containing axes annotations for each of the plot axes suitable for printing to the screen, ``display_iax`` is a cell array containing axes annotations for each of the integration axes suitable for printing to the screen, and ``energy_axis`` gives the index of the column in the 4x4 matrix din.u that corresponds to the energy axis.
+The output returned provides (if such information exists) a vector ``[title_main, title_pax, title_iax, display_pax,
+display_iax, energy_axis]`` where ``title_main`` is the title that would appear at the top of the plot, ``title_pax``
+contains the annotations for each of the plot axes, ``title_iax`` contains the legend detailing the integration axes
+limits etc, ``display_pax`` is a cell array containing axes annotations for each of the plot axes suitable for printing
+to the screen, ``display_iax`` is a cell array containing axes annotations for each of the integration axes suitable for
+printing to the screen, and ``energy_axis`` gives the index of the column in the 4x4 matrix din.u that corresponds to
+the energy axis.
 
 
 Reformatting the data
 =====================
 
-You can convert an sqw object into a dnd object (i.e. you throw away the individual pixel information) quite easily. You simply type
+You can convert an sqw object into a dnd object (i.e. you throw away the individual pixel information) quite easily. You
+simply type
 
 ::
 
    dnd_out=dnd(sqw_obj);
 
 
-If you do not know the number of dimensions of the object ``sqw_obj``. If you do know the number of dimensions (e.g. 2) then you can type
+If you do not know the number of dimensions of the object ``sqw_obj``. If you do know the number of dimensions (e.g. 2)
+then you can type
 
 ::
 
    d2d_out=d2d(sqw_obj);
 
 
-One can also reformat a dnd object so that it is turned into an sqw object, although the pixel information will be empty. This is done by typing
+One can also reformat a dnd object so that it is turned into an sqw object, although the pixel information will be
+empty. This is done by typing
```

Comment:
Is it worth saying what is the effect of leaving the pixel info out? With Alex' new structure, an sqw virtually "is" a dnd without pixels, so maybe recheck this